### PR TITLE
[2.3] Merge unset calls.

### PR DESCRIPTION
**Pull request by SpacePossum**
| Q | A |
| --- | --- |
| Bug fix? | no |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| License | MIT |

To see if a new fixer would make it to SF level.
For ref:  https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/1662

### CODE REVIEW

File: src/Symfony/Component/DependencyInjection/Container.php [hunk 1]
```diff
@@ -311,8 +311,7 @@ public function get($id, $invalidBehavior = self::EXCEPTION_ON_INVALID_REFERENCE
             try {
                 $service = $this->$method();
             } catch (\Exception $e) {
-                unset($this->loading[$id]);
-                unset($this->services[$id]);
+                unset($this->loading[$id],$this->services[$id]);
```
Comment (gharlan) [inline src/Symfony/Component/DependencyInjection/Container.php]: missing a space after `,`

Comment (SpacePossum) [inline src/Symfony/Component/DependencyInjection/Container.php]: indeed ... we need a fixer for that ;)
thanks, will update!

**Comment by dunglas** [comment]
What is the rationale behind merging `unset` calls? It makes the code harder to read IMO.

**Comment by fabpot** [comment]
I agree with @dunglas. No value, closing.

**Comment by SpacePossum** [comment]
Thanks for the feedback :+1: 

FYI @dunglas it is a micro optimization, not wanting it because of readability is fine by me, now I know where the fixer stands :)

(btw. there are already merged `unset` calls in the project at the moment)

**Comment by fabpot** [comment]
I think there is no point in enforcing one style. Depending on the situation one or the other is just fine.

**Comment by SpacePossum** [comment]
perfect :)  
as stated, this PR was just a way to test the waters to figure out on what level the fixer should be (it will be contrib).

**Pull request closed without merging**